lib/hooks.dart

```diff
@@ -396,3 +396,54 @@ T useContext<T>(Context<T> context) => ContextHelpers.unjsifyNewContext(React.us
 ///
 /// Learn more: <https://reactjs.org/docs/hooks-reference.html#useref>.
 Ref<T> useRef<T>([T initialValue]) => Ref.useRefInit(initialValue);
+
+/// Customizes the [ref] value that is exposed to parent components when using [forwardRef] by setting [ref.current]
+/// to the return value of [createHandle].
+///
+/// In most cases, imperative code using refs should be avoided.
```

Comment:
React's hook docs don't do this, but how would you feel about linking to the ref documentation here to describe when they _should_ use refs?
```suggestion
/// In most cases, imperative code using refs should be avoided.
/// > For more information, see <https://reactjs.org/docs/refs-and-the-dom.html>.
```

Reply:
Good idea!